keywords/LiteServBase.py

```diff
@@ -52,8 +52,11 @@ def _verify_not_running(self):
         log_r(resp)
         raise LiteServError("There should be no service running on the port")
 
-    def _wait_until_reachable(self):
-        url = "http://{}:{}".format(self.host, self.port)
+    def _wait_until_reachable(self, port=None):
+        if not port:
```

Comment:
The LiteServ iOS starts off with the default admin port 59850 and public port 49850, which is not the same as the --liteserv-port we specify on the command line. Switching to the new port specified on the command line happens through a REST API call to /start. The _wait_until_reachable takes the port specified on the command line to check for the status as soon as the LiteServ is launched. This will obviously time out as the /start REST API call happens much later, when the tests actually run. This change enables checking the status with any port. The default behavior(self.port) is not changed.